test/Microsoft.Data.Sqlite.Tests/SqliteParameterTest.cs

```diff
@@ -392,6 +394,64 @@ public void Bind_with_prefixed_names()
             }
         }
         
+        [Fact]
+        public void Bind_DateTime_with_Arabic_Culture()
+        {
+            Thread.CurrentThread.CurrentCulture = CultureInfo.GetCultureInfo("ar-SA");
```

Comment:
This needs to be undone at the end of the test. See [this sample](https://github.com/xunit/samples.xunit/tree/master/UseCulture) for a better way.

Reply:
Done.